cmd/client/main.go

```diff
@@ -82,7 +82,7 @@ func main() {
 
 	err := app.Run(os.Args)
 	if err != nil {
-		panic(err)
+		fmt.Printf("an error was found while executing the command. Err: [%s] \n", err)
```

Comment:
print on stderr
exit with a non-0 exit code

Reply:
Yes, you are right. Thanks!